src/site/xdoc/newsarchive.xml

```diff
@@ -36,6 +36,30 @@
 
    <subsection name='2016'>
 
+     <a name="20160901"><h4>Sep 2016 - Apache James Server 3.0</h4></a>
+     <p>We are currently working on the release!!!</p>
+     <p>Keep an eye on the news, we are planning to release the next version in November.</p>
+
+     <a name="20160702"><h4>July 2016 - JMAP implementation</h4></a>
+     <p>Full text search via JMAP.</p>
+     <p>Full text search via the <i>getMessageList</i> JMAP command is now supported.</p>
+     <p>The JMAP search <b>filtering</b> parameter is converted to an ElasticSearch query; then this query is processed by the ElasticSearch backend.</p>
+
+     <a name="20160701"><h4>July 2016 - JMAP implementation</h4></a>
+     <p>Read and Send message with attachments.</p>
+     <p>You can now send and read messages with attachments thru the JMAP commands <i>setMessages</i> and <i>getMessages</i>.</p>
```

Comment:
I think "thru" should not be used in formal presentation. Prefer trough. (same in other places of this PR)